app/views/support/CountryGroupOps.scala

```diff
@@ -0,0 +1,16 @@
+package views.support
+
+import com.gu.i18n.CountryGroup
+import model.SupportPhoneDetails
+
+/**
```

Comment:
Annoyingly IntelliJ adds this by default (It is easy to disable..). Can we please lose the comment ;)?